### PR TITLE
Simplified searchspace into list of parameters

### PR DESCRIPTION
**Pull request by hampusnasstrom**

### CODE REVIEW

**Comment by hampusnasstrom** [comment]
@lauri-codes I got the notebook running as expected. We should probably do the same to the objective(s) or what do you think?

File: src/nomad_bayesian_optimization/schema_packages/bayesian_optimization.py [hunk 1]
```diff
@@ -201,8 +231,53 @@ def from_baybe_campaign(campaign):
...
+                        'm_def': (
+                            'nomad_bayesian_optimization.schema_packages.'
+                            'bayesian_optimization.ContinuousParameter'
+                        ),
+                        'name': parameter['name'],
+                        'lower_bound': parameter['bounds']['lower'],
+                        'upper_bound': parameter['bounds']['upper'],
+                    }
+                )
+            elif parameter['type'] == 'SubstanceParameter':
```
Comment (lauri-codes) [inline src/nomad_bayesian_optimization/schema_packages/bayesian_optimization.py:277]: You could do a small simplification inside the loop, something like this:

```python
for parameter in ps:
    type = parameter['type']
    if type == 'CategoricalParameter':
        parameter = {
            'm_def': 'nomad_bayesian_optimization.schema_packages.bayesian_optimization.CategoricalParameter',
            'name': parameter['name'],
            'values': parameter['values'],
        }
    elif type == ...
    parameters.append(parameter)
```

**Review by lauri-codes — COMMENTED** [review]
Looks good, I had one minor comment.

I will merge my changes and continue a bit on connecting these parameters to the steps. The same exact thing could indeed be done to the objectives: if you want to take a look, youfeel free to prepare another MR.

**Comment by lauri-codes** [comment]
I will just go ahead and merge this. The small updates we can do later.

**Pull request merged**